pkg/service/operation/service.go

```diff
@@ -43,19 +47,51 @@ func (s Service) GetOperations(request GetOperationsRequest) (*GetOperationsResp
 	}
 	for i, op := range ops {
 		op := op
-		newOp := Operation{
-			ID:   op.ID,
-			Done: op.Done,
-			Result: Result{
-				Error:    op.Error,
-				Response: op.Response,
-			},
-		}
+		newOp, _ := serviceModel(op)
 		resp.Operations[i] = newOp
 	}
 	return resp, nil
 }
 
+type ServiceModelFunc func(any) any
+
+func serviceModel(op opstorage.StoredOperation) (Operation, error) {
+	newOp := Operation{
+		ID:   op.ID,
+		Done: op.Done,
+		Result: Result{
+			Error: op.Error,
+		},
+	}
+
+	if len(op.Response) > 0 {
+		switch {
+		case strings.Contains(op.ID, "submissions"):
```

Comment:
could this be an operation constant somewhere to avoid collisions in the future?